src/logic/textBox.js

```diff
@@ -0,0 +1,72 @@
+import { answers, getAnswer} from './answer.js';
+import { getJoke, jokes } from './tellMeJoke';
+import { getFact, facts } from './tellMeFact';
+
+const displayTextBox = (text) => {
+  const Paragraph = document.createElement("p");
+  const questionParagraph = document.createElement("p");
+  const answerOutput = document.getElementById("answerOutput");
+ 
+
+  
+  questionParagraph.textContent = question;
+
+  Paragraph.innerHTML = text;
+
+  questionParagraph.classList.add("question-text");
+  Paragraph.classList.add("answer-text");
+  answerOutput.appendChild(Paragraph);
+  answerOutput.appendChild(questionParagraph);
+};
+
+
+const answerOutput = () => {
+  const questionInput = document.getElementById("questionInput");
+  const questionText = questionInput.value;
+  questionInput.value = "";
+
+  const randomAnswer = getAnswer(answers);
+  showAnswerForQuestion(questionText, randomAnswer);
+};
+
+
+export const initTextBoxBehavior = () => {
+  const jokeButton = document.getElementById("jokeButton");
+  const factButton = document.getElementById("factButton");
+  const questionInput = document.getElementById("questionInput");
+
+  
+
+  jokeButton.addEventListener("click", () => {
+    const randomJoke = getJoke(jokes);
+    displayTextBox(randomJoke);
+  });
+
+  factButton.addEventListener("click", () => {
+    const randomFact = getFact(facts);
+    displayTextBox(randomFact);
+  });
+
+
+  windows.addEventListener("keyup", (event) => {
+    if (event.key === "Enter") {
+      answerOutput();
+    }});
+
+
+    // window.addEventListener("wheel", (event) => {
```

Comment:
Is this code going to be used in the future?

Reply:
This was potential logic for scroll down effect, but i realize this is not the best for user and I changed the concept to scroll down button. This feature is in progress, that's why in the new commit you can see commented code there.
( I make sure to leave a description like you point out in other conversation )